src/components/JournalEntriesView.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { Dispatch } from 'react';
+import { connect } from 'react-redux';
+import { Action } from 'redux';
+import { createStyles, WithStyles, withStyles } from '@mui/styles';
+import { TdrState } from 'reducers';
+import { JournalEntryModel, IamResourceTypeEnum } from 'generated/tdr';
+import { getJournalEntries } from 'actions/index';
+import JournalEntryTable from './table/JournalEntryTable';
+
+const styles = () =>
+  createStyles({
+    className: {
+      color: 'red',
+    },
+  } as const);
```

Comment:
It's my understanding that this pattern is now [considered legacy](https://mui.com/system/styles/basics/) and we're going to need to migrate away from it in order to update material UI. I think we'll need to use the [mui system components](https://mui.com/system/getting-started/overview/) and styled-components moving forward.

Reply:
Ok, I'll take a look at these documents! In the meanwhile, this can be deleted, since they arent being used

Reply:
perfect!